Monika After Story/game/script-topics.rpy

```diff
@@ -3460,15 +3457,15 @@ label monika_surprise:
     m 3f "I kind of felt obligated to do it."
     m 3c "You know what? Maybe I should do it again..."
     m 1b "Yeah, that's a good idea."
-    $ _mas_surprise()
-    if mas_curr_affection_group == mas_aff.G_HAPPY:
+    $ mas_surprise()
+    if mas_curr_affection_group == mas_affection.G_HAPPY:
```

Comment:
at the topic of this file, I imported the mas_affection store as `mas_aff` so its easier to type.

Reply:
When I ran it, it crashes because mas_aff wasn't defined.